modules/local/payload/resources/usr/bin/main.py

```diff
@@ -70,6 +70,8 @@ def rename_file(f, payload, seq_experiment_analysis_dict, date_str):
         variant_type = 'snv'
```

Comment:
This is not correct because `short_variant` contains both `snv` and `indel`. It's not correct to name both snv and indel with `snv` in the file name.

Reply:
Shall we have `snv-indel` instead of `snv`